_partials/_hypercore_policy_workflow.md

```diff
@@ -0,0 +1,57 @@
+1. **Enable columnstore**
+
+   * [Use `ALTER TABLE` for a hypertable][alter_table_hypercore]
+     ```sql
+     ALTER TABLE stocks_real_time SET (timescaledb.enable_columnstore = true, timescaledb.segmentby = 'symbol');
+     ```
+   * [Use ALTER MATERIALIZED VIEW for a continuous aggregate][compression_continuous-aggregate]
+     ```sql
+     ALTER MATERIALIZED VIEW stock_candlestick_daily set (timescaledb.enable_columnstore = true, timescaledb.segmentby = 'symbol' );
+     ```
+
+1. **Add a policy to move chunks to the columnstore at a specific time interval**
+
+   For example, 60 days after the data was added to the table:
+   ``` sql
+   SELECT add_columnstore_policy('older_stock_prices', after => INTERVAL '60d');
```

Comment:
I believe the columnstore API is now using procedures, which use the CALL command instead of SELECT

```suggestion
   CALL add_columnstore_policy('older_stock_prices', after => INTERVAL '60d');
```